packages/react-native-editor/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Can this LICENSE file be safely removed?

Reply:
Removed it in favor of the repo's own one 👍